examples/custom_search_method/local_search_runner/README.md

```diff
@@ -0,0 +1,33 @@
+# Custom SearchMethod with LocalSearchRunner
+
+This example shows how to implement a custom hyperparameter SearchMethod that enables fault tolerance.
+To run the custom SearchMethod, in this example we use LocalSearchRunner that executes the custom SearchMethod 
+on your local machine. 
+For an example of running the custom SearchMethod on a cluster, see `examples/custom_searcher/core_search_runner`.
+
```

Comment:
See my comment in core_search_runner/README.md about conceptually separating the 2 experiments.